doc/gettingstarted.md

```diff
@@ -91,319 +97,555 @@ $ ttn-lw-cli login
 
 It is also possible to register an ABP activated device using the `--abp` flag as follows:
 
 ```bash
-$ ttn-lw-cli end-devices create app1 dev1 --frequency-plan-id EU_863_870 --lorawan-phy-version 1.0.2-b --lorawan-version 1.0.2 --abp --session.dev-addr 00E4304D --session.keys.app-s-key.key A0CAD5A30036DBE03096EB67CA975BAA --session.keys.f_nwk_s_int_key.key B7F3E161BC9D4388E6C788A0C547F255
+$ ttn-lw-cli end-devices create app1 dev2 \
+  --frequency-plan-id EU_863_870 \
+  --lorawan-version 1.0.2 \
+  --lorawan-phy-version 1.0.2-b \
+  --abp \
+  --session.dev-addr 00E4304D \
+  --session.keys.app-s-key.key A0CAD5A30036DBE03096EB67CA975BAA \
+  --session.keys.nwk-s-key.key B7F3E161BC9D4388E6C788A0C547F255
 ```
 
-This will create an LoRaWAN 1.0.2 end device `dev1` with DevAddr `00E4304D`, AppSKey `A0CAD5A30036DBE03096EB67CA975BAA` and NwkSKey `B7F3E161BC9D4388E6C788A0C547F255`.
-
 ## <a name="linkappserver">Linking the application</a>
 
-In order to send uplinks and receive downlinks from your device, you must first link the application server to the network server. In order to do this, first create an API key for the application server:
+In order to send uplinks and receive downlinks from your device, you must link the Application Server to the Network Server. In order to do this, create an API key for the Application Server:
 
 ```bash
-$ ttn-lw-cli app api-keys create --application-id app1 --right-application-link
+$ ttn-lw-cli applications api-keys create \
+  --application-id app1 \
+  --right-application-link
 ```
 
-The CLI will return an API key such as `NNSXS.VEEBURF3KR77ZR...`. This API key has only link rights and can therefore only be used during the linking process. 
+The CLI will return an API key such as `NNSXS.VEEBURF3KR77ZR...`. This API key has only link rights and can therefore only be used for linking.
 
-You can now link the application server to the network server:
+You can now link the Application Server to the Network Server:
 
 ```bash
-$ ttn-lw-cli app link set app1 --api-key NNSXS.VEEBURF3KR77ZR..
+$ ttn-lw-cli applications link set app1 --api-key NNSXS.VEEBURF3KR77ZR..
 ```
 
-Your application is now linked, and can use the built-in MQTT broker and webhooks support to receive uplink traffic and send downlink traffic.
+Your application is now linked. You can now use the builtin MQTT server and webhooks to receive uplink traffic and send downlink traffic.
 
-## <a name="mqtt">Using the MQTT broker</a>
+## <a name="mqtt">Using the MQTT server</a>
 
-In order to use the MQTT broker it is necessary to register a new API key that will be used during the authentication process:
+In order to use the MQTT server you need to create a new API key to authenticate:
 
 ```bash
-$ ttn-lw-cli app api-keys create --application-id app1 --right-application-traffic-down-write --right-application-traffic-read
+$ ttn-lw-cli applications api-keys create \
```

Comment:
Again a `--name`?

Reply:
And perhaps explain that if they start building realy applications they may want to give more rights to the API key, and that they can use `--help` to see which rights they can give an application API key

Reply:
But that they can also do that later with the `applications api-keys update` command

Reply:
Addressed, except last; too much detail for GS